datasets/ph/most_wanted/crawler.py

```diff
@@ -1,22 +1,67 @@
 import csv
 from typing import Dict
-from zavod import Context
+
+from zavod import Context, helpers as h
+
+
+def parse_name(context, name):
+    """
+    Always returns: (full_name, first_name, last_name, suffix)
+    - Two commas: last, first middle, suffix
+    - One comma: last, first middle
+    - No comma: everything is full_name, rest blank
+    """
+    parts = [p.strip() for p in name.split(",")]
+
+    if len(parts) == 3:
+        last_name, first_name, suffix = parts
+        full_name = f"{first_name} {last_name} {suffix}".strip()
+    elif len(parts) == 2:
+        last_name, first_name = parts
+        suffix = ""
+        full_name = f"{first_name} {last_name}".strip()
```

Comment:
I feel we should return `None` for the non-available components here. I think it should be `h.apply_name`'s job to generate the full name.